components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.enrich.ui/src/main/resources/web/enrich-mediator/update-mediator.jsp

```diff
@@ -35,12 +39,23 @@
     if (request.getParameter("mediator.enrich.source.val_ex") != null && !request.getParameter("mediator.enrich.source.val_ex").trim().equals("")) {
 
         if (request.getParameter("mediator.enrich.source.type2").equals("custom")) {
-            XPathFactory xPathFactory = XPathFactory.getInstance();
-            enrichMediator.setSourceExpression(xPathFactory.createSynapseXPath("mediator.enrich.source.val_ex",
-                    request.getParameter("mediator.enrich.source.val_ex"), session));
+            String expression = request.getParameter("mediator.enrich.source.val_ex");
```

Comment:
Can we define a constant for "mediator.enrich.source.val_ex" and use it everywhere?

Reply:
+1 added a constant file